libs/utils/utils.js

```diff
@@ -826,10 +826,13 @@ export async function loadArea(area = document) {
     const loaded = section.blocks.map((block) => loadBlock(block));
     areaBlocks.push(...section.blocks);
 
+    await decorateIcons(section.el, config);
+
     // Only move on to the next section when all blocks are loaded.
     await Promise.all(loaded);
 
-    await decorateIcons(section.el, config);
+    const lcpLoadedEvent = new Event('milo:LCP:loaded');
+    window.dispatchEvent(lcpLoadedEvent);
```

Comment:
Nit: since you're using `lcpLoadedEvent` just once, you could combine the two lines, avoid allocating memory for a variable and trim a few characters. Since this is utils, every little bit counts :)